.github/workflows/atm-run-model-integration-tests.yml

```diff
@@ -0,0 +1,12 @@
+name: ATM Run Model Integration Tests
```

Comment:
These should eventually run on PRs, so I think we can drop the `-run` from the name.

Reply:
Could you drop it from the file name too?  Thanks.

Reply:
done